packages/paste-core/components/chat-log/__tests__/customization.spec.tsx

```diff
@@ -28,175 +29,295 @@ const customBubbleStyles: PasteCustomCSS = {
+      expect(outboundMessage).toHaveStyleRule('margin-left', '2.25rem');
+    });
 
-  it('should set custom element data attribute on ChatBubbles', () => {
-    render(
-      <CustomizationProvider baseTheme="default" theme={TestTheme} elements={customizedFooElements}>
-        <ChatMessage variant="inbound">
-          <ChatBubble element="foo_bubble" data-testid="inbound-bubble">
+    it('should set custom element data attribute', () => {
+      render(
+        <CustomizationProvider baseTheme="default" theme={TestTheme} elements={customizedFooElements}>
+          <ChatMessage element="foo_message" variant="inbound" data-testid="inbound-message">
             test
-          </ChatBubble>
-        </ChatMessage>
-        <ChatMessage variant="outbound">
-          <ChatBubble element="foo_bubble" data-testid="outbound-bubble">
+          </ChatMessage>
+          <ChatMessage element="foo_message" variant="outbound" data-testid="outbound-message">
             test
-          </ChatBubble>
-        </ChatMessage>
-      </CustomizationProvider>
-    );
+          </ChatMessage>
+        </CustomizationProvider>
+      );
+      const inboundMessage = screen.getByTestId('inbound-message');
+      expect(inboundMessage.getAttribute('data-paste-element')).toEqual('foo_message');
+
+      const outboundMessage = screen.getByTestId('outbound-message');
+      expect(outboundMessage.getAttribute('data-paste-element')).toEqual('foo_message');
+    });
+  });
+
+  describe('ChatBubble', () => {
+    it('should add custom styles to variants', () => {
+      render(
+        <CustomizationProvider baseTheme="default" theme={TestTheme} elements={customizedElements}>
+          <ChatMessage variant="inbound">
+            <ChatBubble data-testid="inbound-bubble">test</ChatBubble>
+          </ChatMessage>
+          <ChatMessage variant="outbound">
+            <ChatBubble data-testid="outbound-bubble">test</ChatBubble>
+          </ChatMessage>
+        </CustomizationProvider>
+      );
+
+      const inboundBubble = screen.getByTestId('inbound-bubble');
+      const outboundBubble = screen.getByTestId('outbound-bubble');
+
+      expect(inboundBubble).toHaveStyleRule('background-color', 'rgb(2,99,224)');
+      expect(inboundBubble).toHaveStyleRule('color', 'rgb(255,255,255)');
+      expect(outboundBubble).toHaveStyleRule('background-color', 'rgb(3,11,93)');
+      expect(outboundBubble).toHaveStyleRule('color', 'rgb(255,255,255)');
+    });
+
+    it('should set element data attribute', () => {
+      render(
+        <CustomizationProvider baseTheme="default" theme={TestTheme} elements={customizedElements}>
+          <ChatMessage variant="inbound">
+            <ChatBubble data-testid="inbound-bubble">test</ChatBubble>
+          </ChatMessage>
+          <ChatMessage variant="outbound">
+            <ChatBubble data-testid="outbound-bubble">test</ChatBubble>
+          </ChatMessage>
+        </CustomizationProvider>
+      );
```

Comment:
ℹ️  Since these customization wrappers get repeated often, we started using the second optional argument of `render` to define them. It's kind of nice because you can just define it once, and use it in each render.

Not a required change, just a fun fact. 😸 

```suggestion
      render(
          <ChatMessage variant="inbound">
            <ChatBubble data-testid="inbound-bubble">test</ChatBubble>
          </ChatMessage>
          <ChatMessage variant="outbound">
            <ChatBubble data-testid="outbound-bubble">test</ChatBubble>
          </ChatMessage>,
          { wrapper: YourWrapperComponent }
      );
```

Reply:
Nice! I didn't realize that was a thing. Will update.